Capstone_EDA.ipynb

```diff
@@ -58,7 +58,7 @@
     "\n",
```

Comment:
<p>Die Directory Abbildung hat keinen Mehrwert für den externen Leser. Die Daten sind bei uns zusätzlich eh nicht mehr in der Form. Das ist unnötiger Balast, dieses Bild sollten aus meiner Sicht mit dem nächsten Update raus.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/davidslabon/Capstone_UnlockingClimateSolutions/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Ja genau. Hau das gern raus. Habe ich nicht dran gedacht.</p>